diff --git a/.config/alacritty/alacritty.toml b/.config/alacritty/alacritty.toml
index a1ef429..9aa4aae 100644
--- a/.config/alacritty/alacritty.toml
+++ b/.config/alacritty/alacritty.toml
@@ -24,7 +24,7 @@ foreground = "0xebdbb2"
 
 [font]
 normal = {family = "SauceCodePro Nerd Font Mono"}
-size = 11.5
+size = 8
 
 [scrolling]
 history = 5000
diff --git a/.config/ranger/plugins/__pycache__/__init__.cpython-313.pyc b/.config/ranger/plugins/__pycache__/__init__.cpython-313.pyc
index db14d3e..61aba82 100644
Binary files a/.config/ranger/plugins/__pycache__/__init__.cpython-313.pyc and b/.config/ranger/plugins/__pycache__/__init__.cpython-313.pyc differ
diff --git a/.config/xmonad/README.org b/.config/xmonad/README.org
index 8361e28..e87542d 100644
--- a/.config/xmonad/README.org
+++ b/.config/xmonad/README.org
@@ -243,7 +243,7 @@ myEventHook = refocusLastEventHook <+> hintsEventHook
 #+begin_src haskell
 main :: IO ()
 main = do
-  xmproc <- spawnPipe "xmobar $XDG_CONFIG_HOME/xmobar/xmobarrc"
+  xmproc <- spawnPipe "xmobar $XDG_CONFIG_HOME/xmobar/laptop-xmobarrc"
   xmonad $ ewmhFullscreen $ addDescrKeys ((mod4Mask, xK_F1), xMessage) myKeys $ docks  def  {
     terminal              = myTerminal
   , modMask               = myModMask
diff --git a/.config/xmonad/xmonad.hs b/.config/xmonad/xmonad.hs
index 99ee8a1..666c427 100644
--- a/.config/xmonad/xmonad.hs
+++ b/.config/xmonad/xmonad.hs
@@ -137,7 +137,7 @@ myTall = renamed [Replace "tall"]
   $ windowNavigation
  -- $ subLayout [] (smartBorders Simplest)
   $ limitWindows 5
-  $ mySpacing 10
+  $ mySpacing 15
   $ ResizableTall 1 (3/100) (1/2) []
 
 myMirror = renamed [Replace "mirror tall"]
@@ -145,7 +145,7 @@ myMirror = renamed [Replace "mirror tall"]
   $ Mirror myTall
 
 myGrid = renamed [Replace "grid"]
-  $ mySpacing 10
+  $ mySpacing 5
   $ limitWindows 4
   $ Grid
 
@@ -208,7 +208,7 @@ myEventHook = refocusLastEventHook <+> hintsEventHook
 
 main :: IO ()
 main = do
-  xmproc <- spawnPipe "xmobar $XDG_CONFIG_HOME/xmobar/xmobarrc"
+  xmproc <- spawnPipe "xmobar $XDG_CONFIG_HOME/xmobar/laptop-xmobarrc"
   xmonad $ ewmhFullscreen $ addDescrKeys ((mod4Mask, xK_F1), xMessage) myKeys $ docks  def  {
     terminal              = myTerminal
   , modMask               = myModMask
